src/App.js

```diff
@@ -1,16 +1,18 @@
 import './App.css';
 import { Outlet } from 'react-router-dom';
 import NavBar from './side_navbar/Navbar';
+import MotorcycleDetails from './pages/MotorcycleDetails';
 
 function App() {
   return (
     <div className="relative min-h-screen md:flex">
       <NavBar />
       <div>
         {/* Put all other content in this div  */}
-        <h1>Motocycle Book</h1>
+        <MotorcycleDetails />
```

Comment:
Optional: For our next iteration it would be nice if we added the <MotorcycleDetails /> as a route in our main 'index.js' so that it only reflects when the user selects a motorcycle detail. As at now it is present in every route

Reply:
@Beardless-sheik, Thank for the comment